src/sentry/static/sentry/app/data/forms/organizationSecurityAndPrivacyGroups.tsx

```diff
@@ -7,7 +7,9 @@ import {
 } from 'app/utils/crashReports';
 import {JsonFormObject} from 'app/views/settings/components/forms/type';
 
-const organizationSecurityAndPrivacy: JsonFormObject[] = [
+// Export route to make these forms searchable by label/help
+export const route = '/settings/:orgId/security-and-privacy/';
```

Comment:
Nice! Can we replicate this for Relay as well? (not in this PR)

Reply:
sure